---a/net/minecraft/world/biome/BiomeGenHills.java
+++b/net/minecraft/world/biome/BiomeGenHills.java
@@ -1,6 +1,7 @@
 package net.minecraft.world.biome;
 
 import java.util.Random;
+
 import net.minecraft.block.BlockSilverfish;
 import net.minecraft.init.Blocks;
 import net.minecraft.util.BlockPos;
@@ -11,8 +11,8 @@
 import net.minecraft.world.gen.feature.WorldGenTaiga2;
 import net.minecraft.world.gen.feature.WorldGenerator;
 
-public class BiomeGenHills extends BiomeGenBase
-{
+public class BiomeGenHills extends BiomeGenBase {
+
     private WorldGenerator theWorldGenerator;
     private WorldGenTaiga2 field_150634_aD;
     private int field_150635_aE;
@@ -21,8 +21,7 @@
     private int field_150638_aH;
     private static final String __OBFID = "CL_00000168";
 
-    protected BiomeGenHills(int p_i45373_1_, boolean p_i45373_2_)
-    {
+    public BiomeGenHills(int p_i45373_1_, boolean p_i45373_2_) {
         super(p_i45373_1_);
         this.theWorldGenerator = new WorldGenMinable(Blocks.monster_egg.getDefaultState().withProperty(BlockSilverfish.VARIANT, BlockSilverfish.EnumType.STONE), 9);
         this.field_150634_aD = new WorldGenTaiga2(false);
@@ -31,41 +31,35 @@
         this.field_150637_aG = 2;
         this.field_150638_aH = this.field_150635_aE;
 
-        if (p_i45373_2_)
-        {
+        if (p_i45373_2_) {
             this.theBiomeDecorator.treesPerChunk = 3;
             this.field_150638_aH = this.field_150636_aF;
         }
     }
 
-    public WorldGenAbstractTree genBigTreeChance(Random p_150567_1_)
-    {
-        return (WorldGenAbstractTree)(p_150567_1_.nextInt(3) > 0 ? this.field_150634_aD : super.genBigTreeChance(p_150567_1_));
+    public WorldGenAbstractTree genBigTreeChance(Random p_150567_1_) {
+        return (WorldGenAbstractTree) (p_150567_1_.nextInt(3) > 0 ? this.field_150634_aD : super.genBigTreeChance(p_150567_1_));
     }
 
-    public void decorate(World worldIn, Random p_180624_2_, BlockPos p_180624_3_)
-    {
+    public void decorate(World worldIn, Random p_180624_2_, BlockPos p_180624_3_) {
         super.decorate(worldIn, p_180624_2_, p_180624_3_);
         int i = 3 + p_180624_2_.nextInt(6);
         int j;
         int k;
         int l;
 
-        for (j = 0; j < i; ++j)
-        {
+        for (j = 0; j < i; ++j) {
             k = p_180624_2_.nextInt(16);
             l = p_180624_2_.nextInt(28) + 4;
             int i1 = p_180624_2_.nextInt(16);
             BlockPos blockpos1 = p_180624_3_.add(k, l, i1);
 
-            if (worldIn.getBlockState(blockpos1).getBlock() == Blocks.stone)
-            {
+            if (worldIn.getBlockState(blockpos1).getBlock() == Blocks.stone) {
                 worldIn.setBlockState(blockpos1, Blocks.emerald_ore.getDefaultState(), 2);
             }
         }
 
-        for (i = 0; i < 7; ++i)
-        {
+        for (i = 0; i < 7; ++i) {
             j = p_180624_2_.nextInt(16);
             k = p_180624_2_.nextInt(64);
             l = p_180624_2_.nextInt(16);
@@ -73,17 +73,14 @@
         }
     }
 
-    public void genTerrainBlocks(World worldIn, Random p_180622_2_, ChunkPrimer p_180622_3_, int p_180622_4_, int p_180622_5_, double p_180622_6_)
-    {
+    public void genTerrainBlocks(World worldIn, Random p_180622_2_, ChunkPrimer p_180622_3_, int p_180622_4_, int p_180622_5_, double p_180622_6_) {
         this.topBlock = Blocks.grass.getDefaultState();
         this.fillerBlock = Blocks.dirt.getDefaultState();
 
-        if ((p_180622_6_ < -1.0D || p_180622_6_ > 2.0D) && this.field_150638_aH == this.field_150637_aG)
-        {
+        if ((p_180622_6_ < -1.0D || p_180622_6_ > 2.0D) && this.field_150638_aH == this.field_150637_aG) {
             this.topBlock = Blocks.gravel.getDefaultState();
             this.fillerBlock = Blocks.gravel.getDefaultState();
-        }
-        else if (p_180622_6_ > 1.0D && this.field_150638_aH != this.field_150636_aF)
-        {
+        } else if (p_180622_6_ > 1.0D && this.field_150638_aH != this.field_150636_aF) {
             this.topBlock = Blocks.stone.getDefaultState();
             this.fillerBlock = Blocks.stone.getDefaultState();
         }
@@ -92,8 +92,7 @@
         this.generateBiomeTerrain(worldIn, p_180622_2_, p_180622_3_, p_180622_4_, p_180622_5_, p_180622_6_);
     }
 
-    private BiomeGenHills mutateHills(BiomeGenBase p_150633_1_)
-    {
+    public BiomeGenHills mutateHills(BiomeGenBase p_150633_1_) {
         this.field_150638_aH = this.field_150637_aG;
         this.func_150557_a(p_150633_1_.color, true);
         this.setBiomeName(p_150633_1_.biomeName + " M");
@@ -102,9 +102,7 @@
         return this;
     }
 
-    protected BiomeGenBase createMutatedBiome(int p_180277_1_)
-    {
+    public BiomeGenBase createMutatedBiome(int p_180277_1_) {
         return (new BiomeGenHills(p_180277_1_, false)).mutateHills(this);
     }
-}
+}
